Segments/Leng.mapproj

```diff
@@ -79351,7 +79822,7 @@
 ]]></block>
         <block><![CDATA[]]></block>
       </script>
-      <entry entity="cliffBossKosh" size="1" minimum="1" maximum="1" />
+      <entry entity="BossLevel11Kosh" size="1" minimum="1" maximum="1" />
```

Comment:
Boss levels seem a little odd. Power Troll is definitely weaker than Kosh but is rated at level 12 to Kosh's 11? Power Troll is for NV before Kosh and key for princess quest key for the stat pot.

Reply:
He's a miniboss. I can reflect the power level differences in a future pull - Some monsters are set to the highest end variable, some have a multiplier of 3 on hit points, +3 attack +3 defense from their basic model, and some have 6x hps and a few other variables - see Drake/Vlad. They're on that format now (Drake didn't change at all essentially. Kosh lost hps)